plugins/ssr/ssrast.mli

```diff
@@ -84,7 +84,7 @@ type ssripat =
   | IPatId of (*TODO id_mod option * *) Id.t
   | IPatAnon of anon_iter (* inaccessible name *)
 (* TODO  | IPatClearMark *)
-  | IPatDispatch of ssripatss (* /[..|..] *)
+  | IPatDispatch of bool (* check number of branches and number of goals match *) * ssripatss (* (..|..) *)
```

Comment:
I'm lost. I thought the discussion below showed that this was not the real meaning of the boolean. Does `false` mean we don't check the number of branches and number of goals match?